extensions/functions_arithmetic.yaml

```diff
@@ -1614,3 +1614,131 @@ window_functions:
         decomposable: NONE
         return: i64?
         window_type: PARTITION
+  - name: "first_value"
+    description: >
+      The `expression` is evaluated against the first row.  The `expression` can be a
+      column, expression or subquery that evaluates to a single value.
```

Comment:
The wording here seems overly verbose to me.  Technically every argument in every function is an expression.

> The `expression` can be a column, expression or subquery that evaluates to a single value.

A column is not a thing in the spec. I suspect you mean a field reference which is an expression.

> or subquery that evaluates to a single value.

This is also an expression.

> The `expression` is evaluated against the first row.

This is not necessarily true in many implementations.  Many naive implementations of `first(x*x)` will first calculate `x*x` on the entire input and then take the first item.  It's a good optimization but I don't think we should imply that it is always done this way.